dbt/adapters/databricks/python_models/python_submissions.py

```diff
@@ -224,7 +223,9 @@ def compile(self, path: str) -> PythonJobDetails:
         if access_control_list:
             job_spec["access_control_list"] = access_control_list
 
-        return PythonJobDetails(self.run_name, job_spec, additional_job_config)
+        return PythonJobDetails(
```

Comment:
For some reason, mypy was complaining that this init was giving too many args when it was a dataclass.  I have no idea what causes that, but switching to a pydantic model fixes the complaint.